docs/gettingstarted/installation-windows.rst

```diff
@@ -98,6 +98,14 @@ All commands need to be performed in either PowerShell or a Command Shell.
 
    Unless configured otherwise the store will use ``saleor`` as both username and password. Remeber to give your user the ``SUPERUSER`` privilege so it can create databases and database extensions.
 
+#. Create a PostgreSQL database
+
+   See `PostgreSQL's createdb command <https://www.postgresql.org/docs/current/static/app-createdb.html>`_ for details.
+
+   .. note::
+
+       Database name is extracted from the ``DATABASE_URL`` environmental variable. If absent it defaults to ``saleor``.
```

Comment:
> **The** database name

> the ``DATABASE_URL`` environmental variable

It's an **environment variable**, it's not an ecological variable.

---

Also, I can't really manage to make much sense to the edits. Why not just just taking or basing your changes on the Linux or MacOS docs?

https://github.com/mirumee/saleor/blob/d9467f967cabe365b50e913e004abd22f8c42a94/docs/gettingstarted/installation-macos.rst#L125-L131

Reply:
@NyanKiyoshi the text is copy pasted from the Linux page.

Reply:
I really think that just redirecting the user to another documentation, just to very (very) implicitly tell them to do `createdb saleor` is a very bad idea. It will confuse many users and make them give up on the installation has they would have to search the information or simply not get the point.

Also, are the postgres binaries in the Windows' `PATH`? Or does the user has to set it/ tell Windows where to look for `createdb`?

Reply:
I don't disagree but this patches a significantly worse issue with the docs and does it quickly without causing problems.